tests/wpt/mozilla/tests/mozilla/structuredclone.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<meta charset="utf-8">
+<title></title>
+<html>
+<head>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+async_test(function(t) {
+  var blob = new Blob(['<a id="a"><b id="b">hey!</b></a>'], {type:"text/plain"});
```

Comment:
1. Why can't this test be submitted into web-platform-tests
2. Is there no test for this yet?